README.md

```diff
@@ -0,0 +1,5 @@
+Welcome to the Recon-ng Marketplace! The official module repository for the Recon-ng Framework.
```

Comment:
Why is the README being affected by this PR?

Reply:
Looks like it was being renamed earlier in the PR.

Reply:
ok, so if it isn't now, then why is it being seen as a different between your branch and the master?

Reply:
I see, it is still trying to at a README to the .github folder. Please remove that.